packages/expo-notifications/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 🛠 Breaking changes
 
+- On Android, fix the `badge count number` not being read from the notification payload content. ([#17171](https://github.com/expo/expo/pull/17171) by [@danstepanov](https://github.com/danstepanov))
```

Comment:
@wschurman, we haven't used such labeling style like `[android]` in the CHANGELOG so far.
Why do we have to change it now? Is there any specific reason backing your suggestion? 🤔 
Moreover, we tend to be very accurate in terms of proper naming (since https://github.com/expo/expo/pull/17134), so it would be `Android` instead of `android`.